AK/StdLibExtras.h

```diff
@@ -125,6 +125,26 @@ constexpr bool is_constant_evaluated()
 #endif
 }
 
+// These can't be exported into the global namespace as they would clash with the C standard library.
+
+#define __DEFINE_GENERIC_ABS(type, zero, intrinsic) \
+    constexpr type abs(type num)                    \
+    {                                               \
+        if (is_constant_evaluated())                \
```

Comment:
Question, should this be:
```suggestion
        if constexpr (is_constant_evaluated())                \
```
(I'm not sure _at all_ if it should be an `if constexpr`, maybe it works without or there's a special case for `is_constant_evaluated()`, but I feel like it should ^^)

Reply:
Nope
`if constexpr (is_constant_evaluated())` is straight up wrong, it's literally equivalent to `if constexpr (true)`.

Reply:
Okay, then you cna disregard this x)